libevmasm/Assembly.cpp

```diff
@@ -681,6 +686,9 @@ LinkerObject const& Assembly::assemble() const
 			ret.bytecode.push_back(uint8_t(Instruction::PUSH20));
 			ret.bytecode.resize(ret.bytecode.size() + 20);
 			break;
+		case AccountAbstractionPrefix:
+			ret.bytecode.insert(ret.bytecode.end(), accountAbstractionPrefix, accountAbstractionPrefix+37);
```

Comment:
Should either hoist `37` into a constant, or maybe `std::size`, though I don't know if that works, or if solidity is C++17.

Reply:
Agreed, something like that would be preferable. I don't have that much C++ experience, so I initially went with the simplest solution. Will look into it.

Reply:
Changed both places where I previously used a hardcoded length to `sizeof()`